src/classes/ERR_Handler.cls

```diff
@@ -188,6 +188,7 @@ public with sharing class ERR_Handler {
 	    	}
 	                
 	    	if(UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
+	    	   UTIL_Debug.debug('**** Storing errorRecords: ' + errorRecords);
```

Comment:
We need to look at the overhead at some point, but youre right, this probably won't be the straw that hurts our camel